public/styles/components/_footer.scss

```diff
@@ -196,24 +192,17 @@
       margin-top: space(6x);
     }
   }
-}
 
-@media (min-width: $breakpoint-lg-min) {
   .bsds-footer {
```

Comment:
can we move above bsds-nav-footer so the parent > child relationship is clear in the css?
also worth checking if this level of specificity is required, or if we can just remove bsds-footer as a wrapper and reduce specificity overall